app/javascript/pages/country/widget/widgets/widget-emissions-deforestation/widget-emissions-deforestation.js

```diff
@@ -0,0 +1,69 @@
+import { createElement, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import COLORS from 'pages/country/data/colors.json';
```

Comment:
If passing in colors from parent this isn't needed.

Reply:
oh you're right I forgot your changes